src/components/SocialIcons.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div id="social">
+    <a
+      href="https://github.com/aecworks"
+      :class="gIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="gHandleExpand"
+      @mouseout="gHandleCollapse"
+    >
+      <img
+        alt="Github"
+        :class="gIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/github.svg"
+      />
+      <div :class="gIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">Github</div>
+    </a>
+    <a
+      href="https://twitter.com/gtalarico"
+      :class="tIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="tHandleExpand"
+      @mouseout="tHandleCollapse"
+    >
+      <img
+        alt="Twitter"
+        :class="tIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/twitter.svg"
+      />
+      <div :class="tIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">Twitter</div>
+    </a>
+    <a
+      href="https://twitter.com/gtalarico"
+      :class="aIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="aHandleExpand"
+      @mouseout="aHandleCollapse"
+    >
+      <img
+        alt="About"
+        :class="aIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/about.svg"
+      />
+      <div :class="aIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">About</div>
+    </a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'SocialIcons',
+  data() {
+    return {
+      gIconExpanded: false,
+      tIconExpanded: false,
+      aIconExpanded: false,
+    }
+  },
+  methods: {
+    gHandleExpand: function () {
+      this.gIconExpanded = true
+    },
+    gHandleCollapse: function () {
+      this.gIconExpanded = false
+    },
+    tHandleExpand: function () {
+      this.tIconExpanded = true
+    },
+    tHandleCollapse: function () {
+      this.tIconExpanded = false
+    },
+    aHandleExpand: function () {
+      this.aIconExpanded = true
+    },
+    aHandleCollapse: function () {
+      this.aIconExpanded = false
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+#social {
+  z-index: 2;
+  position: fixed;
+  display: flex;
+  @include for-large-up {
+    right: 0;
+    top: 50vh;
+    margin-right: 1rem;
+    flex-direction: column;
+    direction: rtl;
+  }
+  @include for-large-down {
+    top: calc(100vh - 32px - 1rem);
+    z-index: 3;
+  }
+}
+
+.transition {
+  width: 32px;
+}
+
+.transition:hover {
+  width: 120px;
+  transition: all 0.1s;
```

Comment:
could we up this to 200ms for consistency?

https://github.com/aecworks/aec.works-web/blob/9e4046cc62b8185d118da1ba430f718ceb2a34d6/src/assets/scss/components/buttons.scss#L27